src/components/Auth/Login.js

```diff
@@ -1,19 +1,177 @@
 import React, { useState, useEffect } from 'react';
 import { useSelector, useDispatch } from 'react-redux';
+import styled from 'styled-components';
 import Switch from 'components/Shared/Switch';
 import StyledButton from 'components/Shared/Button';
-import PwdInput from 'components/Auth/Shared/PwdInput';
-import IdInput from 'components/Auth/Shared/IdInput';
 import * as S from 'components/Auth/styles';
 import { login } from '../../store/auth';
 import { useNavigate } from 'react-router';
 
+const LoginContainer = styled.div`
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+`;
+
+const LoginOptionContainer = styled.div`
+  margin-bottom: 40px;
+  border-bottom: 1px solid #eee;

```

Comment:
ThemeProvider 얘기가 나왔던거 같은데!  혹시 어떻게 진행되고 있는지 알 수 있을까요?
사용하신다면 디자인 팀이 지정한 color schema 에 맞게 지정하고 사용할 수 있어서 편리합니다!
음... 너무 하라는 압박처럼 느껴질수도 있을 것 같아 죄송하네요 해당 PR 까지만 언급하고 넘어가겠습니당 ㅎㅎ...

```suggestion
  border-bottom: 1px solid ${({theme}) => theme.color.borderColor};
```